packages/amplify-provider-awscloudformation/resources/rootStackTemplate.json

```diff
@@ -26,6 +26,53 @@
         }
       }
     },
+    "DeploymentBucketBlockHTTP": {
+      "Type": "AWS::S3::BucketPolicy",
+      "Properties": {
+        "Bucket": {
+          "Ref": "DeploymentBucketName"
```

Comment:
should this ref be the bucket name or the bucket itself? (There may be no difference if the default ref value for buckets is the name)

Reply:
we parameterize it, and reference the parameter
![image](https://github.com/user-attachments/assets/b380eea3-ff42-41ff-8fac-e6c0bf1f2f62)